### ISSUE TITLE
Alpha/mu

### ISSUE THREAD

**Issue by ccsgonc** [wpgp/bottom-up-tutorial#5]
Should be mu here, right?
https://github.com/wpgp/bottom-up-tutorial/blob/f626e886d56d091cbb3dbfbf476675d1686fabaf/tutorials/tutorial1/tutorial1_linearRegression.Rmd#L1150

**Comment by edarin** [comment]
correct, thanks! the [stored model](https://github.com/wpgp/bottom-up-tutorial/blob/f626e886d56d091cbb3dbfbf476675d1686fabaf/tutorials/tutorial2/tutorial2_model2.stan) is correct.
